wsgi.py

```diff
@@ -0,0 +1,2 @@
+from mcgj import create_app
```

Comment:
Did you mean to check this file in?

Reply:
Yeah, this allows local usage of `flask run`. Is there a different way to do it?

Reply:
Yep! Just set the `FLASK_APP` to `mcgj` in your environment (either in the .env file or in your shell), then in the root of the repo, run `python3 -m flask run`.